src/components/layout/header/language-switch.tsx

```diff
@@ -25,15 +25,18 @@ const StyledSelection = styled.select`
 
   color: inherit;
   padding-left: 18px;
+  text-align: right;
+  /*"direction: rlt" aligns the text right in Safari*/
+  direction: rtl;
```

Comment:
This looks hacky and when I disable them I see minimal changes but for sure not what I would expect when dealing with `direction` and `rtl`. I'm sure there is a better solution. Can you speak about your general issues you had here?

Reply:
The problem is that the `text-align: right` doesn't work in Safari. I did a quick research and found the solution with `direction`, which would work here.

Screenshot without `direction: rtl` in Safari
<img width="159" alt="Bildschirmfoto 2022-03-10 um 08 12 00" src="https://user-images.githubusercontent.com/57712895/157610424-935ddc6f-e66e-4125-895b-aa8c202e00fe.png">